irohad/consensus/yac/storage/impl/yac_vote_storage.cpp

```diff
@@ -77,10 +82,37 @@ namespace iroha {
             break;
           case ProposalState::kSentNotProcessed:
             val = ProposalState::kSentProcessed;
-            break;
           case ProposalState::kSentProcessed:
-            break;
+            last_sent_processed_proposal_round_ =
```

Comment:
Seems that we have to pass out only committed rounds.

Reply:
realized in 2ac4575